tools/image_processing/fractal/cellvoyager_to_ome_zarr_init.xml

```diff
@@ -0,0 +1,152 @@
+<tool id="fractal_cellvoyager_to_ome_zarr" name="CellVoyager image to OME-Zarr" version="1.4.3" profile="24.2">
+    <description>Create OME-Zarr images from Cellvoyager images</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="aggressive">
+    <![CDATA[
+        mkdir -p tmp/input_images tmp/zarr_output tmp/outjson && 
+
+        #for $i, $filename in enumerate($input_images):
+            ln -s '$filename' 'tmp/input_images/${filename.element_identifier}' &&
+        #end for
+
+        version=`python3 --version | awk '{print tolower($1) substr($2,1,4)}'` &&
+        path=`which python3` &&
+        base="\${path%/*/*}" &&
+        new_path="\$base/lib/\$version/site-packages/fractal_tasks_core/tasks" &&
+        init_execute="cellvoyager_to_ome_zarr_init.py" &&
+        cli_init="\$new_path/\$init_execute" &&
+       
+        python3 \$cli_init --args-json '${init_params}' --out-json ./outout.json &&
+		    
+        compute_execute="cellvoyager_to_ome_zarr_compute.py" &&
+        cli_compute="\$new_path/\$compute_execute" &&
+		
+        python3 $__tool_directory__/split_add_params.py --init_json ./outout.json --extra_params $compute_extra_params --outdir tmp/outjson &&
+        
+	
+        parallel -j \${GALAXY_SLOTS:-1} python3 \$cli_compute --args-json {} --out-json {.}_compute.json ::: tmp/outjson/entry_*.json  &&
+
+        ##maybe use the below for tests until Galaxy support the test of zarr ?
```

Comment:
we could do that, but I would only execute this when some user-hidden parameter is set (hidden="true"), this param can be activated in tests.

Reply:
and you should test `$zarr_plate.extra_files_path`